queries/projects/lup-projects.sql

```diff
@@ -86,15 +97,15 @@ SELECT
         'hearing_location', disp.dcp_publichearinglocation,
         'vote_date', disp.dcp_dateofvote,
         'vote_location', disp.dcp_votelocation,
-        -- 'vote_infavor', disp.dcp_votinginfavorrecommendation,
-        -- 'vote_against', disp.dcp_votingagainstrecommendation,
-        -- 'vote_abstained', disp.dcp_votingabstainingonrecommendation,
-        -- 'vote_total_appointed', disp.dcp_totalmembersappointedtotheboard,
+        'vote_infavor', disp.dcp_votinginfavorrecommendation,
+        'vote_against', disp.dcp_votingagainstrecommendation,
+        'vote_abstained', disp.dcp_votingabstainingonrecommendation,
+        'vote_total_appointed', disp.dcp_totalmembersappointedtotheboard,
         'vote_quorum', disp.dcp_wasaquorumpresent,
         'rec_bb', disp.dcp_boroughboardrecommendation,
         'rec_cb', disp.dcp_communityboardrecommendation,
         'rec_bp', disp.dcp_boroughpresidentrecommendation
-        -- 'rec_comment', disp.dcp_consideration
+        'rec_comment', disp.dcp_consideration
```

Comment:
i know these were uncommented and were existing there before but we should keep the column names the same